spec/rubocop/cop/performance/double_start_end_with_spec.rb

```diff
@@ -0,0 +1,89 @@
+# encoding: utf-8
+
+require 'spec_helper'
+
+describe RuboCop::Cop::Performance::DoubleStartEndWith do
+  subject(:cop) { described_class.new }
+
+  before do
+    inspect_source(cop, source)
+  end
+
+  context 'two #start_with? calls' do
+    context 'with the same receiver' do
+      context 'all parameters of the second call are pure' do
+        let(:source) { 'x.start_with?(a, b) || x.start_with?("c", D)' }
```

Comment:
`pure` doesn't mean "has not side effect when evaluated". It means that the same input always maps to the same output, that the only input is parameters (no global, instance vars, etc) and that they are no side effects. I'll check the definition.